BootloaderCommonPkg/Library/PagingLib/PagingMap.c

```diff
@@ -98,23 +98,10 @@ MapMemoryRange (
     }
     PageTable4K = (UINTN *)((UINTN)PageBuffer + PteOffset);
     PageTable2M = (UINTN *)((UINTN)PageBuffer + PdeOffset);
-    break;
-  case SIZE_1GB:
-    ASSERT (Ranges[0].Limit < SIZE_512GB);
-    PageBits    = GetPageShiftBits (PDP_PG_LVL);
-    PdpOffset   = 1 * EFI_PAGE_SIZE;
-    EntryNum    = 512;
-    PageTable1G = (UINTN *)((UINTN)PageBuffer + PdpOffset);
-    break;
-  default:
-    return EFI_INVALID_PARAMETER;
-  }

```

Comment:
The changes looks good to me.
Had better remove the lines 69 ~ 71: They should be initialized to NULL instead 0,  and now it looks they are not required.

Reply:
Yes, agree. let me make that change.